transformers/component_tests/queries/flap_bid_event_computed_columns_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
 	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
 	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore"
```

Comment:
I made a note in  https://github.com/makerdao/vulcanizedb/pull/96 as well, but I'm not sure I understand the switch to importing `datastore` when we're also still importing `repositories` on the line below. Not a big deal, just wanted to understand what's going on! Also, super possible that I'm having "end of the day before a day off brain". ;p

Reply:
Yeah the idea is that for the type we import the interface from `datastore` and then when we initialize we use `NewHeaderRepository` from `repositories`, but the actual implementation of `headerRepository` in `repositories` is private